packages/@aws-cdk/aws-neptune-alpha/lib/cluster.ts

```diff
@@ -332,6 +332,13 @@ export interface DatabaseClusterProps {
    * @default - false
    */
   readonly copyTagsToSnapshot?: boolean;
+
+  /**
+   * The port number
```

Comment:
It is better to be more helpful.

```suggestion
   * The port number on which the DB instances in the DB cluster accept connections.
```